test/models/medium_test.rb

```diff
@@ -0,0 +1,124 @@
+require "test_helper"
+
+describe Medium do
+
+  describe 'validations' do
+
+    before do
+      @medium = Medium.new(
+        title: 'CyberBabies on High',
+        description: 'cannot recommend this book in good conscience',
+        category: 'book',
+        creator: 'Jarth Dengees',
+        publication_year: 1999
+      )
```

Comment:
Why not use an existing medium saved in the fixtures? (I definitely think that this is very readable though, so I won't fight for this very terribly, but the fixtures are definitely there already.)